cpp/arcticdb/version/test/rapidcheck_version_map.cpp

```diff
@@ -235,6 +235,8 @@ struct GetAllVersions : rc::state::Command<Model, MapStorePair> {
 };
 
 RC_GTEST_PROP(VersionMap, Rapidcheck, ()) {
+    char env[] = "ARCTICDB_NO_STRICT_SYMBOL_CHECK=0";
+    putenv(env);
```

Comment:
Should reset this environment variable after the tests to avoid affecting a subsequent test.

We also have a run-time `ConfigsMap` which can be configured from environment variables (via a Python hook) and is easier to reset in tests.